src/smart-app-banner.scss

```diff
@@ -0,0 +1,319 @@
+//colors
```

Comment:
How `scss` relate to better workflow?
Not sure we can accept that change — no one is going to maintain scss files.